README.adoc

```diff
@@ -1695,8 +1696,8 @@ describe Article do
   describe '#title' do
     it 'is unique' do
       another_article = FactoryBot.create(:article, title: article.title)
-      article.valid?
-      expect(article.errors[:title].size).to eq(1)
+      another_article.valid?
+      expect(another_article.errors[:title].size).to eq(1)
```

Comment:
This implies there's a
```ruby
let(:article) { FactoryBot.create(:article) }
```
somewhere, and the expectation is made on that original article.
As per the balancing example, that makes sure it's valid when there's no "another" article that the original article is `valid?`.